src/tests/class_tests/openms/source/SvmTheoreticalSpectrumGenerator_test.cpp

```diff
@@ -95,7 +95,13 @@ START_SECTION(void simulate(PeakSpectrum &spectrum, const AASequence &peptide, b
   PeakMap exp;
   MzMLFile mz_file;
 
+#if OPENMS_BOOST_VERSION_MINOR < 56
   mz_file.load(OPENMS_GET_TEST_DATA_PATH("SvmTheoreticalSpectrumGenerator_test.mzML"),exp);
+  TEST_EQUAL(spec.size(), 7);
+#else
+  mz_file.load(OPENMS_GET_TEST_DATA_PATH("SvmTheoreticalSpectrumGenerator_test_boost58.mzML"),exp);
+  TEST_EQUAL(spec.size(), 8);
+#endif
```

Comment:
hmm that is also possible as I am building against Ubuntu 16.04 -- however the function call clearly uses a RNG while it uses a pre-trained model so my bet would still be on the boost RNG